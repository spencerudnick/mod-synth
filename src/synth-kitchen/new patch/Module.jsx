import React from 'react';

export default function Module({ id }) {
	return (
		<>
			<h1>{id}</h1>
			<button className="delete">Delete</button>
		</>
	);
}
