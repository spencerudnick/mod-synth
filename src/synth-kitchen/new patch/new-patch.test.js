import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import NewPatch from './new-patch';
import Module from './Module';
configure({ adapter: new Adapter() });

it('renders without crashing', () => {
	shallow(<Module />);
});

it('<Module /> has <h1>', () => {
	const wrapper = shallow(<Module />);
	expect(wrapper.find('h1')).toHaveLength(1);
});

it('<Module /> has delete button', () => {
	const wrapper = shallow(<Module />);
	expect(wrapper.find('button.delete')).toHaveLength(1);
});

// test('Module has a prop id 1', () => {
// 	const module = renderer.create(<Module id="module-1" />);
// 	let tree = module.toJSON()
// 	expect(tree).
// });
