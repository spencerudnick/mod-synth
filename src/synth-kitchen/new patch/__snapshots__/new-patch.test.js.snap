// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Module has a prop id 1`] = `
<div
  id="module-1"
/>
`;
